lib/faraday/request.rb

```diff
@@ -9,6 +10,18 @@ module Faraday
   #     req.body = 'abc'
   #   end
   #
+  # @!attribute method
+  #   @return [String] the HTTP method of the Request
+  # @!attribute path
+  #   @return [URI, String] the path
+  # @!attribute params
+  #   @return [Object] query parameters
+  # @!attribute headers
+  #   @return [Object] headers
+  # @!attribute body
+  #   @return [Object] body
+  # @!attribute options
+  #   @return [Object] options
```

Comment:
It would be great to give the exact type for each of these attributes rather than just `Object`.
I understand this would mean doing some runtime debugging, so I'm happy to remove these for now if that's too much and we can always improve this later (there are some "improve doc" tickets in the v1.0 project already)

Reply:
I learned what they were by adding a `.tap {|x| p(x.options.class.name) }` in the `.create` method.